### PR TITLE
hm trigger

### PR DESCRIPTION
**Pull request by aortizve**

### CODE REVIEW

**Comment by aortizve** [comment]
Hello @mpuccio, I have implemented a basic multiplicity filter. The local compilation is ok. Can you please have a look and help me to understand why the checks are not successful? Thanks!

**Comment by mario-krueger** [comment]
you need to run `clang-format -i` on your .cxx and .h files so they match the code formatting conventions of O2

File: EventFiltering/PWGMM/multFilter.cxx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+		multiplicity.add("hdNdeta", "dNdeta", HistType::kTH1F, {{50,-2.5,2.5, " "}});
+		multiplicity.add("fLeadingTrackPt", "pT of high pT tracks", HistType::kTH1F, {{150, 0., +150., "track #it{p}_{T} (GeV/#it{c})"}});
+		multiplicity.add("fTrackMult", "charged particle multiplicity", HistType::kTH1F, {{200, -0.5, +199.5, "number of tracks (|#eta|<0.8)"}});
+
+		multiplicity.add("fLeadingTrackPtSelected", "pT of selected high pT tracks", HistType::kTH1F, {{150, 0., +150., "track #it{p}_{T} (GeV/#it{c})"}});
+		multiplicity.add("fTrackMultSelected", "charged particle multiplicity of the selected events", HistType::kTH1F, {{200, -0.5, +199.5, "number of tracks (|#eta|<0.8)"}});
+
+		std::array<std::string, 2> eventTitles = {"all", "rejected"};
+
+		auto scalers{std::get<std::shared_ptr<TH1>>(multiplicity.add("fProcessedEvents", "Multiplicity - event filtered;;events", HistType::kTH1F, {{kNtriggersMM+2, -0.5, kNtriggersMM + 2 - 0.5}}))};
```
Comment (mario-krueger) [inline EventFiltering/PWGMM/multFilter.cxx:72]: `std::get<std::shared_ptr<TH1>>(multiplicity.add("fProcessedEvents", "Multiplicity - event filtered;;events", HistType::kTH1F, {{kNtriggersMM+2, -0.5, kNtriggersMM + 2 - 0.5}}))`
can be replaced by
`multiplicity.add<TH1>("fProcessedEvents", "Multiplicity - event filtered;;events", HistType::kTH1F, {{kNtriggersMM+2, -0.5, kNtriggersMM + 2 - 0.5}})`
This is not only shorter, but also more save, as it handles correctly the case when the histogram is not found.

File: EventFiltering/PWGMM/multFilter.cxx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+		for (size_t iBin = 0; iBin < eventTitles.size()+mmObjectsNames.size(); iBin++) {
+			if(iBin<2)
+				scalers->GetXaxis()->SetBinLabel(iBin + 1, eventTitles[iBin].data());
+			else
+				scalers->GetXaxis()->SetBinLabel(iBin + 1, mmObjectsNames[iBin-2].data());
```
Comment (mario-krueger) [inline EventFiltering/PWGMM/multFilter.cxx:77]: I think single line `if` statements without `{}` are not allowed. You will be asked to add these 'readability braces' by the CI.

**Comment by mpuccio** [comment]
Hi @aortizve! Thanks a lot for the first contribution! The check failing is clang-format, an automatic tool to ensure that we have uniform code formatting across the repository. To fix this, you can do from your repository directory: `git-clang-format upstream/master`, this will create the necessary modifications to pass the formatting step (you will need to commit them and then push here). Let me know if you need any further help. When that is fixed I will approve to see the compilation going and then we can discuss tomorrow afternoon the changes.

File: EventFiltering/CMakeLists.txt [hunk 3]
```diff
@@ -1,54 +1,56 @@
-# Copyright 2019-2020 CERN and copyright holders of ALICE O2.
-# See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
-# All rights not expressly granted are reserved.
+#Copyright 2019 - 2020 CERN and copyright holders of ALICE O2.
```
Comment (mpuccio) [inline EventFiltering/CMakeLists.txt:1]: It appears that git-clang-format is overzealous here. Can you exclude the changes from git-clang-format to the cmake files?

**Review by mpuccio — CHANGES_REQUESTED** [review]

File: EventFiltering/CMakeLists.txt (hunk 3 above)
Comment (aortizve) [reply EventFiltering/CMakeLists.txt]: ok, done

**Review by mpuccio — APPROVED** [review]

**Comment by mpuccio** [comment]
Thanks! I now approve to get it in the tag for tomorrow.

**Pull request merged**